src/TranscriptionAnalyzer.ts

```diff
@@ -30,28 +30,47 @@ export class TranscriptionAnalyzer implements ITranscriptionAnalyzer {
      * We prefer to throw rather than making a best guess at resolving typos.
      */
     public validateExpectedTranscription = (expectedTranscription: string): void => {
-        if (this.expectedRegEx.test(expectedTranscription)) {
+        if (this.uncleanTranscriptionRegEx.test(expectedTranscription)) {
             console.log(colors.red(
                 `Error on expected transcription: "${expectedTranscription}"\n`));
 
-            const message = `Transcriptions may only contain letters, numbers, apostrophes, and spaces.`;
+            const message = `Transcriptions may only contain letters, apostrophes, and spaces.`;
             throw SyntaxError(message);
         }
     };
 
     /**
-     * The STT service will not return hyphens, commas, periods, etc. Expect
-     * that special characters in transcription files match the STT output.
+     * Actual and expected transcriptions should be cleaned in the same way, but
+     * only when we know exactly the behavior we are cleaning.
      *
-     * Some other oddities as we find them: `okay` is transcribed as `OK`. `you`
-     *    is sometimes transcribed as `ya`. Apostrophes mid-word are sometimes
-     *    preceded by a space.
+     * We still do not know exactly what to expect from the STT service, but
+     * that uncertainty will be handled in `analyzeActualTranscription`.
      */
-    public cleanExpectedTranscription = (expectedTranscription: string): string => {
-        return expectedTranscription
-            .replace(/-/g, ` `)
-            .replace(/\bokay\b/g, `ok`)
-            .toLowerCase();
+    public cleanTranscription = (transcription: string): string => {
+        return transcription
+            .toLowerCase()
+            .replace(/\bokay\b/g, `ok`);
```

Comment:
Please consider creating a separate config fille where we can list the replacement pairs. That way, this entire code doesn't need to be recompiled if we decide to add another replacement pair.

This could be a future consideration.

Reply:
I won't have time today to work on that, but I've created https://github.com/Joll59/d-ser-t/issues/38 for whenever we get to it.